src/wrappers/React.js

```diff
@@ -24,12 +24,12 @@ const makeReactContainer = Component => {
     }
 
     render () {
-      const { children, ...rest } = this.state
+      const { children, '': invoker, ...rest } = this.state
```

Comment:
What's this `'': invoker`?

Reply:
Thanks for asking - I actually thought you might know, as I thought it was coming from vuera.

I believe it's coming from vue attaching an event handler, but somehow it is missing an event name, so it ends up using an empty string. Related vue code: https://github.com/vuejs/vue/blob/53b77f805475f217eb221a475409e9a27f590549/src/core/vdom/helpers/merge-hook.js#L37.

Reply:
Interesting. Are you sure this cannot break anything? :)
Anyway, we should leave a comment explaining the situation near this line.

Reply:
Can we leave a comment there? :)